Prox/Prox/Data/PlacesController.swift

```diff
@@ -30,32 +30,22 @@ class PlacesProvider {
         return RemoteConfigKeys.searchRadiusInKm.value
     }()
 
-    fileprivate var displayedPlaces = [Place]() {
-        didSet {
-            var placesMap = [String: Int]()
-            for (index, place) in displayedPlaces.enumerated() {
-                placesMap[place.id] = index
-            }
-            self.placeKeyMap = placesMap
-        }
-    }
-    fileprivate var placeKeyMap = [String: Int]()
+    private var allPlaces = [Place]()
 
+    fileprivate var displayedPlaces = [Place]()
```

Comment:
I agree, but didn't bother here since I noticed that `PlacesProvider` was already doing sorting -- and I figured this PR was already big enough without more refactoring :)